.github/workflows/eslint.yml

```diff
@@ -0,0 +1,53 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+# ESLint is a tool for identifying and reporting on patterns
+# found in ECMAScript/JavaScript code.
+# More details at https://github.com/eslint/eslint
+# and https://eslint.org
+
+name: ESLint
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
+
+jobs:
+  eslint:
+    name: Run eslint scanning
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      security-events: write
+      actions: read # only required for a private repository by github/codeql-action/upload-sarif to get the Action run status
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+          # We must fetch at least the immediate parents so that if this is
+          # a pull request then we can checkout the head.
+        with:
+          fetch-depth: 2
+
+      # If this run was triggered by a pull request event, then checkout
+      # the head of the pull request instead of the merge commit.
+      - run: git checkout HEAD^2
+        if: ${{ github.event_name == 'pull_request' }}
+
+      - name: Install ESLint
+        run: |
+          npm install eslint@8.57.0
```

Comment:
we can just use `npm ci` to install the same eslint version, as we use in our `package.json` + it'll also install eslint plugins which we use for linting (like `typescript-eslint`)

and let's also use `--ignore-scripts` flag, so `npm pretest` defined in `postinstall` won't be executed (as it's not needed for `eslint`) - https://github.com/FlowCrypt/flowcrypt-browser/blob/764daf24cbd7b855ef06888305dd76f45483f7a0/.github/workflows/eslint.yml#L41

Reply:
Ok - I will check and do the necessary changes. Will get back to you soon.